src/pages/Login/Signup.components.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```

Comment:
Why did you name Signup.components.jsx? Signup is a page. So change a name to Signup.jsx

Reply:
Ok thanks for replying

Reply:
My bad, this isn't a full review yet.